invenio_oauthclient/views/settings.py

```diff
@@ -79,7 +79,15 @@ def index():
     # Sort according to title
     services.sort(key=itemgetter('title'))
 
+    # Check if local login is possible
+    local_login_enabled = current_app.config.get(
+        "ACCOUNTS_LOCAL_LOGIN_ENABLED", True
+    )
+    password_set = current_user.password is not None
+    local_login_possible = local_login_enabled and password_set
+
     return render_template(
         'invenio_oauthclient/settings/index.html',
-        services=services
+        services=services,
+        local_login_impossible=not local_login_possible,
```

Comment:
i've added this kind of inverted logic here because a falsy value for this parameter (e.g. by leaving it out) would produce a result that's most in line with how everything worked before (no restriction of rendering on any "disconnect" buttons)

Reply:
Agree, I would maybe have changed the name of the var to something like `only_external_login`.